contracts/errors/ETHTransferFail.sol

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.14;
```

Comment:
do you think we should make the files in contracts-libs `^0.8.17`

Reply:
Yes

Reply:
Let's do in another PR